CONTRIBUTING.md

```diff
@@ -148,6 +148,7 @@ repository](https://github.com/zulip/zulip/issues?q=is%3Aopen+is%3Aissue+label%3
 - Avoid issues with the "difficult" label unless you
   understand why it is difficult and are highly confident you can resolve the
   issue correctly and completely.
+- Look for issues labled as "good first issue" as these are typically lower difficulty
 
```

Comment:
Sure, we could say something like this. Please modify it to:

> -  Take a look at issues with the ["good first issue" label](https://github.com/zulip/zulip/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), as they are especially accessible to new contributors. However, you will likely find issues without this label that are accessible as well.

Also, let's make this the second bullet on the list.

Reply:
@alya , sounds great!